components/library/zeromq/zeromq5/zeromq.p5m

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Shouldn't you remove zeromq3/zeromq.p5m ?
Also, shouldn't we depend on ALL zeromq versions to be respectful to packages dependent on older library/c++/zeromq?